tombstones.go

```diff
@@ -119,6 +123,8 @@ func writeTombstoneFile(logger log.Logger, dir string, tr TombstoneReader) error
 		return err
 	}
 	f = nil
+	meta.Stats.NumBytes += size
```

Comment:
This can result in higher `NumBytes` than it is actually on disk. 

Consider that `sizeof(stone1)` was already in `NumBytes`, and with another `Delete()`, `stone2` was added to the set of tombstones. In that case the tombstone file is written fresh, so the writes contain `stone1` and `stone2`. So incrementing like this would count `sizeof(stone1) + sizeof(stone1) + sizeof(stone2)`.

Reply:
correct, will refactor.

Reply:
:+1: